src/accordion/accordion.spec.ts

```diff
@@ -18,7 +18,15 @@ function getPanels(element: Element): Element[] {
   return Array.from(element.querySelectorAll('ngb-accordion-group .panel'));
 }
 
-function hasTitle(element: Element, str: string): Boolean {
+function getOpenState(panels: Element[]): boolean[] {
+  return panels.map((panel) => { return hasClass(panel, 'panel-open'); });
```

Comment:
Yes. `toHaveCssClass` is a matcher (AKA usable with `expect`) not a method on `Element`.